build/mbl/build-update-payloads.py

```diff
@@ -89,14 +89,14 @@ def main():
     for command in bitbake_build_commands:
         bitbake.run_command(command, check=True)
 
-    # Create the payloads
+    # Create the payloads for bootloader components
     bootloader1_base_path = args.outputdir / "bootloader1_payload"
     bootloader2_base_path = args.outputdir / "bootloader2_payload"
     kernel_base_path = args.outputdir / "kernel_payload"
     rootfs_base_path = args.outputdir / "rootfs_payload"
     multi_component_base_path = args.outputdir / "multi_component_payload"
 
-    create_update_payload_commands = [
+    create_update_bootloader_payloads_commands = [
```

Comment:
Why did you change the name of this function? We are still creating update payloads for the kernel, rfs and multi-components.

Reply:
This is a name of the list of the commands we use for creating bootloader payloads. I've been more explicit.

Reply:
But we are not only creating bootloader components update payloads.

Reply:
Exactly, this list contains ONLY the bootloader components.

Reply:
Looks like it also contains the rootfs and kernel, which aren't bootloader components.

Reply:
OK, I see the update if also for rootfs and kernel.

Reply:
Fixed